src/modules/esl-event-listener/core/targets/resize.adapter.ts

```diff
@@ -1,6 +1,8 @@
 import {SyntheticEventTarget} from '../../../esl-utils/dom/events/target';
 import {ESLElementResizeEvent} from './resize.adapter.event';
 
+export {ESLElementResizeEvent};
```

Comment:
do we really need this export?

Reply:
Yep, it is functional.
Otherwise, we will need to mention the events file in the module cumulative file, but we can consider this option as well
Do you voting for moving export to `src/modules/esl-event-listener/core.ts` ?